<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProgrammePrincipalQtClass</class>
 <widget class="QMainWindow" name="ProgrammePrincipalQtClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>600</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Fenêtre principale du programme de gestion des élections</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QWidget" name="verticalLayoutWidget">
    <property name="geometry">
     <rect>
      <x>510</x>
      <y>210</y>
      <width>217</width>
      <height>128</height>
     </rect>
    </property>
    <layout class="QVBoxLayout" name="verticalLayout">
     <item>
      <widget class="QPushButton" name="inscriptionCandidat">
       <property name="text">
        <string>Inscription d'un candidat</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="inscriptionElecteur">
       <property name="text">
        <string>Inscription d'un électeur</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="desincriptionPersonne">
       <property name="text">
        <string>Désinscription d'une personne</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="quitter">
       <property name="text">
        <string>Quitter (pour terminer l'application)</string>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
   <widget class="QLabel" name="titrePrincipal">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>60</y>
      <width>621</width>
      <height>81</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <family>Nimbus Sans L</family>
      <pointsize>18</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Bienvenue sur le programme de gestion des élections</string>
    </property>
    <property name="textFormat">
     <enum>Qt::PlainText</enum>
    </property>
   </widget>
   <widget class="QTableWidget" name="tableWidget">
    <property name="geometry">
     <rect>
      <x>90</x>
      <y>150</y>
      <width>351</width>
      <height>291</height>
     </rect>
    </property>
    <property name="columnCount">
     <number>3</number>
    </property>
    <column>
     <property name="text">
      <string>Nom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Prénom</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Date de Naissance</string>
     </property>
    </column>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>27</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuMenu">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="actionQuitter"/>
   </widget>
   <addaction name="menuMenu"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionQuitter">
   <property name="text">
    <string>Quitter</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>quitter</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammePrincipalQtClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>360</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>actionQuitter</sender>
   <signal>triggered()</signal>
   <receiver>ProgrammePrincipalQtClass</receiver>
   <slot>close()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>-1</x>
     <y>-1</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inscriptionCandidat</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammePrincipalQtClass</receiver>
   <slot>dialogCandidat()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>378</x>
     <y>261</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>inscriptionElecteur</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammePrincipalQtClass</receiver>
   <slot>dialogElecteur()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>284</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>desincriptionPersonne</sender>
   <signal>clicked()</signal>
   <receiver>ProgrammePrincipalQtClass</receiver>
   <slot>dialogSupprimerPersonne()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>618</x>
     <y>317</y>
    </hint>
    <hint type="destinationlabel">
     <x>399</x>
     <y>299</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>dialogCandidat()</slot>
  <slot>dialogElecteur()</slot>
  <slot>dialogSupprimerPersonne()</slot>
 </slots>
</ui>
